src/platform/capslock_osx.cpp

```diff
@@ -3,11 +3,14 @@
  * Copyright © 2024 The TokTok team.
  */
 
-#include "src/platform/capslock.h"
+#include "capslock.h"
```

Comment:
You have changed `#include "src/platform/capslock.h"` to `#include "capslock.h"` in here (`src/platform/capslock_osx.cpp`), but kept it unchanged in `src/platform/capslock_win.cpp` and `src/platform/capslock_x11.cpp`.

Reply:
Thanks, done.

The reason for this change is that later I want to use clang-format's header reordering. It needs the "current module" header to be in the current directory and a relative include.

Reply:
It supports absolute includes as well. It's how I use it in in some of my projects. :P You can also set it up to bear the naming of your test files in mind when sorting to make sure that the header you're testing functions from is sorted at the top: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includeismainregex